.github/workflows/ci.yml

```
@@ -14,9 +14,9 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-        - '2.7'
-        - '3.5'
+        - '3.6'
         - '3.7'
+        - '3.8'
```

This is that suggestion I made in the comment above.

```suggestion
        - '3.6'
```

---

3.10 is on the following line, after this diff.

---

The alternative is to do every Python version, "3.6, 3.7, 3.8, 3.9, 3.10". Either we keep all the Python versions in the supported range or we do min-max, but just excluding 3.9 is weird.

---

Oh I missed that Python 3.9 was skipped. IMO the CI takes less than 1 minute per Python at the moment, so I personally lean towards the "test explicitly every Python you claim to support". But at the same time I don't have any real problem to min-max-ing it.

So I'll defer to others, but if it was me I'd do 3.6, 3.7, 3.8, 3.9, 3.10.

---

Okay, I'll do all the Python versions. Since they're fast, anyway!